packaging/os/homebrew.py

```diff
@@ -37,6 +37,11 @@
             - name of package to install/remove
         required: false
         default: None
+    path:
+        description:
+            - ':' separated list of paths to search for 'brew' executable. Since A package (I(formula) in homebrew parlance) location is prefixed relative to the actual path of I(brew) command, providing an alternative I(brew) path enables managing different set of packages in an alternative location in the system.
+        required: false
+        default: '/usr/local/bin'
```

Comment:
I don't consider it so because the parameter `path` existed earlier. It just wasn't explicitly added in the `DOCUMENTATION` block and thus never showed up in the module documentation.